crates/oxc_linter/src/rules/eslint/no_console.rs

```diff
@@ -10,8 +10,17 @@ use crate::{
     rule::Rule,
 };
 
-fn no_console_diagnostic(span: Span) -> OxcDiagnostic {
-    OxcDiagnostic::warn("eslint(no-console): Unexpected console statement.")
+fn no_console_diagnostic(span: Span, allow: &[CompactStr]) -> OxcDiagnostic {
+    let only_msg = if allow.is_empty() {
+        String::new()
+    } else {
+        format!(
+            "Only supported methods: {}",
+            allow.iter().map(ToString::to_string).collect::<Vec<String>>().join(", ")
```

Comment:
```suggestion
            allow.iter().map(oxc_span::CompactStr::as_str).join(",")
```
we can make this a bit better - we don't need to `collect` or convert to `Strings` before joining